catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/pom.xml

```diff
@@ -192,7 +196,7 @@
                             filter-v_1_1_0,
                             ows-v_1_0_0,
                             xlink-v_1_0,
-                            jaxb2-basics-runtime;version="1.11.1",
+                            jaxb2-basics-runtime;version="0.11.0",
```

Comment:
❓Do you remember why we had to go back to a different version?

Reply:
1.11.1 is not a real version, it was released on accident and we apparently used it on accident as well. Had we actually looked at maven central, we most likely would have seen in all caps "1.11.1-RELEASED-ON-ACCIDENT-DONT-USE" and here we are using it :)

Reply:
Looks like `1.11.1` was published by mistake: https://maven-repository.com/artifact/org.jvnet.jaxb2_commons/jaxb2-basics-runtime.